README.md

```diff
@@ -0,0 +1,407 @@
+***
+<p>
+  <a href="https://github.com/k0sm0naft/java-online-book-store" target="_blank">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
+  </a>
+  <a href="https://github.com/k0sm0naft/java-online-book-store" target="_blank">
+    <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-yellow.svg" />
+  </a>
+</p>
+<h1 align="center"> 📚 Welcome to Book Nook App</h1>
+<div align="center">
+  <img src="img/title.png" alt="Book Nook App"/>
+</div>
+
+> In the digital age, online bookstores have revolutionized how we buy and sell books, offering unparalleled convenience and accessibility. **Book Nook App** was inspired by the need for a seamless, user-friendly backend platform where book lovers can discover, purchase, and manage their favorite reads.
+>
+> **Book Nook App** addresses key challenges:
+> - **For Customers**: Simplifies finding the right book with robust search and filter options, ensuring a smooth purchasing experience through an efficient order management system.
+> - **For Administrators**: Provides comprehensive tools for inventory management, order tracking, and real-time updates, making it easy to manage an online bookstore.
+> 
+> Built with Spring Boot, Spring Security, Spring Data JPA, and Springdoc OpenAPI, **Book Nook App** offers a secure and robust backend, perfect for supporting the operations of an online bookstore.
+
+***
+
+## 📜 Table of Contents
+
+- [▶Demo](#Demo)
+- [▶Technologies Used](#Technologies-Used)
+- [▶Features](#Features)
+    - [▶Swagger Documentation](#Swagger-Documentation)
+    - [▶Authentication](#Authentication)
+    - [▶Book](#Book)
+    - [▶Category](#Category)
+    - [▶Shopping Cart](#Shopping-Cart)
+    - [▶Order](#Order)
+- [▶Short Overview](#Short-Overview)
+- [▶Setup Instructions](#Setup-Instructions)
+    - [▶Required](#Required)
+    - [▶Installation](#Installation)
+    - [▶Running Tests](#Running-Tests)
+- [▶Postman Collection](#Postman-Collection)
+- [▶Challenges Faced](#Challenges-Faced)
+- [▶Author](#Author)
+- [▶License](#License)
+
+***
+
+## ✨Demo
+
+### How to use demo:
+
+<details><summary>Use Swagger UI</summary>
+
+1. Open [link](http://ec2-54-234-35-71.compute-1.amazonaws.com/api/swagger-ui/index.html).
```

Comment:
I have checked your swagger, manager login and category creation works fine but you have 500 error on `GET /books` & `GET /categories`, we should fix that